t/admin/plugins.t

```diff
@@ -162,3 +162,65 @@ plugins:
     }
 --- no_error_log
 [error]
+
+
+
+=== TEST 8: get all the attributes of all plugins
+--- config
+    location /t {
+        content_by_lua_block {
+            local json = require("toolkit.json")
+            local t = require("lib.test_admin").test
+
+            local code, message, res = t('/apisix/admin/plugins/?all=true',
+                ngx.HTTP_GET
+            )
+
+            if code >= 300 then
+                ngx.status = code
+                ngx.say(message)
+                return
+            end
+
+            res = json.decode(res)
+            ngx.say(json.encode(res))
+        }
+    }
+--- response_body eval
+qr/\{"name":"example-plugin","priority":0,"schema":\{"properties":\{"i":\{"minimum":0,"type":"number"\},"ip":\{"type":"string"\},"port":\{"type":"integer"\},"s":\{"type":"string"\},"t":\{"minItems":1,"type":"array"\}\},"required":\["i"\],"type":"object"\},"type":"other","version":0.1\}/
+--- no_error_log
+[error]
+
+
+
+=== TEST 9: confirm the plugin of auth type
```

Comment:
more test cases: `priority`, `schema`, `consumer_schema`

Reply:
add a test case for `consumer_schema`.
update the title of test case no.8 in `t/admin/plugins.t`, the `name`, `priority`, `schema`,`type` and `version` of plugin has covered by test case no.8.

Reply:
or as you say more test cases means?